updated readme and removed managed notebook command

karticn-google opened the pull request.

statmike commented:
Hello @karticn-google,
Thank you for getting the automation side of this project going!  

One change I would recommend before we continue here.  I notice that both options in the readme.md suggest copy/pasting the command from the .sh file.  In this case the .sh file is not actually being used as a script.  For simplicity it may be better to put the example command in-line in the readme using a code block like
```
gcloud ...
```

Thank You,
Mike

karticn-google commented:
Hey @statmike,
I have deleted the `user_managed_notebook.sh` script and added the **sample command** to `readme file`. The step to replace the values in the placeholder has been updated as well. Thanks for your feedback.

Regards,
Kartic

statmike commented:
@karticn-google Thank You!  I merged it in
